server/src/controllers/userController.js

```diff
@@ -14,20 +15,20 @@ exports.register = async function (req, res) {
       email,
       password,
     });
-    res
-      .status(200)
-      .json({ message: `You've successfully registered ${user.firstName}` });
+    res.status(200).json({
+      message: `You've successfully registered ${user.firstName}`,
+      name: user.firstName,
+    });
   } catch (err) {
     //handle registration errors
-    console.log(err);
     res.status(400).json({ message: err.errors });
   }
 };
 exports.login = async function (req, res) {
-  console.log(req.body);
   const { email, password } = req.body;
   try {
     const user = await User.findOne({ where: { email } });
+    const { UUID, firstName } = user;
     //check if user doesn't exist in db
     //render register page
     if (user === null) {
```

Comment:
Will `user` always be null if it doesn't exist in the DB? I'm wondering if there's a more robust way to check.